metadata.yaml

```diff
@@ -1,23 +1,16 @@
-# Copyright 2021 Canonical Ltd.
+# Copyright 2022 Canonical Ltd.
 # See LICENSE file for licensing details.
 
-# For a complete list of supported options, see:
-# https://juju.is/docs/sdk/metadata-reference
-name: template-operator
+name: mysql
+display-name: MySQL
 description: |
-  TEMPLATE-TODO: fill out the charm's description
+  MySQL charm  for machines
 summary: |
-  TEMPLATE-TODO: fill out the charm's summary
+  MySQL is  a widely used, open-source relational database management system
+  (RDBMS). MySQL InnoDB cluster provides a complete high availability solution
+  for MySQL via Group Replication.provides
```

Comment:
There's a lost `provides` here

Reply:
removed in cc37f2fa62a8ccfb339e75b1fd51ab0c8977f1d7